index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title></title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css"href="css-index.css">
+  </head>
+  <body background="pexels-aleksandar-pasaric-2603464.jpg">
+    <div class="container">
+      <nav class="navegation">
+        <div class="name">Hello Iam RamonMg</div>
+        <div class="buttons">
+          <button>Formacion</button>
```

Comment:
Training or Education instead of Formacion

Reply:
agreed corrected